Extensions/Effects/old-film-pixi-filter.js

```diff
@@ -0,0 +1,42 @@
+gdjs.PixiFiltersTools.registerFilterCreator('OldFilm', {
+  makePIXIFilter: function(layer, effectData) {
+    var oldFilmFilter = new PIXI.filters.OldFilmFilter();
+
+    return oldFilmFilter;
+  },
+  update: function(filter, layer) {},
```

Comment:
this is where you would handle the filter's animated properties.
I believe you need to set seed to a random value here. Check how its done on the pixi-filter's example

Reply:
Thank you! That works well!